### PR TITLE
Fix typo on homepage and adjust text padding

### LINKED ISSUE #543: Text is not have padding
image

Text should be aligned propel and ensure ease for the reader.

There is how it should look like.

image

### LINKED ISSUE #547: Minor Home Page Typo
I've noticed a small typo on the homepage, where "communinity" is written instead of "community."

While this is a minor issue, correcting it could help maintain the project’s professional appearance in terms of content, especially for uninitiated users. I’d be happy to make this correction if that’s helpful!

image

### PR DESCRIPTION
**Pull request by mohanamisra**
This pull request makes the following changes:

- Corrects a typo on the homepage by changing "communinity" to "community" (resolves #547).
- Adds padding adjustments to enhance text readability (resolves #543).

Do me know if any additional changes are required.
Thank you!

### CODE REVIEW

File: index.html
Comment (therealharshit) [inline index.html:1]: I think it should be community and not <community></community>

**Comment by therealharshit** [comment]
@mohanamisra i think it should be community and not <community></community> which you have added.

**Comment by mohanamisra** [comment]
Faux pas on my side honestly. Should really pay more attention to auto-complete, my bad !

**Comment by therealharshit** [comment]
@mohanamisra your commit messages are also not justifying the changes you made.

**Comment by mohanamisra** [comment]
@therealharshit Could you clarify what you mean?

**Comment by therealharshit** [comment]
> @therealharshit Could you clarify what you mean?

@mohanamisra your this d70ede11c9b5ec5e8f169624d9352f20877fc1c3 commit says it "Fixed homepage typo" but i think it fixed padding issue, and vice-versa for f1c025e6b9823cb3af357030c3e15ee5d46f5540.

**Comment by mohanamisra** [comment]
I apologise for that, @therealharshit . I have revised the commit message errors while the file changes remain the same.

**Comment by therealharshit** [comment]
> I apologise for that, @therealharshit . I have revised the commit message errors while the file changes remain the same.

Please there is no need to apologize, you can join https://matrix.to/#/!OllliPClazQIIzhwoF:matrix.org?via=matrix.org where we have discussion related to this project.

**Comment by quozl** [comment]
Your change to custom.css adds text without a terminating newline.  Please fix.

Also, your commit messages are okay, but weak.  Please read our guidance at https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits

**Comment by mohanamisra** [comment]
@quozl Fixed the lack of a newline, and tried to adhere to the guidelines for the new commit 👍🏽

**Pull request merged**